src/BLE_Common.cpp

```diff
@@ -56,7 +56,7 @@ void BLECommunications(void *pvParameters)
                                 {
                                     logBufP += sprintf(logBufP, "%02x ", pData[i]);
                                 }
-                                logBufP += sprintf(logBufP, "<- %s | %s", myAdvertisedDevice.serviceUUID.toString().c_str(), myAdvertisedDevice.charUUID.toString().c_str());
+                                logBufP += sprintf(logBufP, "<- %s | %s", myAdvertisedDevice.serviceUUID.toString().substr(0,8).c_str(), myAdvertisedDevice.charUUID.toString().substr(0,8).c_str());
```

Comment:
> Why only print the first 8?

Because I don't think we can shorten the uuid's for echelon or flywheel (because they are custom and not registered) and since they are so long they are polluting the logs. We should be able to identify them in our debug however by just their first 8.

Reply:
> Also, if needed, rather than creating a new string use the format specifier `%.8s`

I'm all for a pro's better way

Reply:
Fixed in 8b8de57b4516ccded004ab05f30a74c487dce49f